app/views/partners/dashboards/show.html.erb

```diff
@@ -1,4 +1,5 @@
 <header class="container mx-auto mb-5">
+  <%= render 'broadcast_announcement' unless @broadcast_announcements.empty? %>
```

Comment:
Use `if x.any?` instead of `unless x.empty?` - positive conditions are always easier to read.